content/sensu-go/5.13/reference/tessen.md

```diff
@@ -92,7 +92,7 @@ default      | `false`
 type         | Boolean
 example      | {{< highlight shell >}}opt_out": false{{< /highlight >}}
 
-## Tessen example
+## Tessen configuration example (#tessen-configuration-example)
```

Comment:
```suggestion
## Tessen configuration example
```

This wasn't working as `#tessen-configuration-example` should actually be surrounded by `{}`. However, it isn't required as long as the ToC part matches the heading regardless of case and `-`.